packages/template-retail-react-app/app/components/display-price/index.jsx

```diff
@@ -26,8 +26,10 @@ import ListPrice from '@salesforce/retail-react-app/app/components/display-price
  * @param quantity - quantity to take into the account for price display
  * @param currentPriceProps - extra props to be passing to CurrentPrice component
  * @param listPriceProps - extra props to be passing to ListPrice component
+ * @param productLabelForA11y - product name for a11y
  */
 const DisplayPrice = ({
+    productLabelForA11y,
```

Comment:
I probably wasn't clear enough last time.. but what I'm trying to avoid here is using the word "product".. reason being is that if there is a possibility that we would use this component for something that isn't directly a product price.. say a cart total or sub total.. and we want to have a prefix label.. it doesn't make sense to be called product something. 

Maybe something like labelForA11yLiveRegion or labelForLiveRegion, something generic enough.

Reply:
`labelForA11y` should suffice since it can be used for different purpose apart from live region of the components. I've pushed the changes. Thanks for the feedback